### PR TITLE
Create New Permission- Calendars.ReadBasic

### PR DESCRIPTION
**Pull request by iamgirishck**
Added information around Calendars.ReadBasic across Delegated and Application permissions

### CODE REVIEW

**Comment by JarbasHorst** [comment]
Hello @iamgirishck,

thanks for your PR. Can you please share with me via Teams a pointer to the CSDL and API.md files related to this PR?

**Comment by iamgirishck** [comment]
@Jarbas-MSFT @JarbasHorst - Can you please share your MSFT alias? I am unable to find you in outlook. Thanks

**Comment by Jarbas-MSFT** [comment]
Hi @iamgirishck, I think you found me in Teams :) Let's continue our chat there.

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -394,6 +394,7 @@ None.
 | _Calendars.Read.Shared_ |Read user and shared calendars |Allows the app to read events in all calendars that the user can access, including delegate and shared calendars. |No | No |
 | _Calendars.ReadWrite_ |Have full access to user calendars |Allows the app to create, read, update, and delete events in user calendars. |No | Yes |
 | _Calendars.ReadWrite.Shared_ |Read and write user and shared calendars |Allows the app to create, read, update and delete events in all calendars the user has permissions to access. This includes delegate and shared calendars.|No | No |
+| _Calendars.ReadBasic_ |Read basic details of user calendars |Allows the app to read events in user calendars, except for properties such as body, attachments, and extensions.|No | No |
```
Comment (JarbasHorst) [inline concepts/permissions-reference.md:397]: According to the permission description in the Azure portal, this permission requires admin consent. If that is the case, please update this "Admin Consent Required" column with the correct value.

image

Comment (JarbasHorst) [inline concepts/permissions-reference.md:397]: Are you assigning these new permissions to a method already?

Comment (iamgirishck) [reply concepts/permissions-reference.md]: I have updated the content

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for a final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**